src/tools/mdbm_export_splitter.cc

```diff
@@ -90,8 +90,7 @@ getNumForHashCode(const char *optarg)
                           string("STL"),      string("MD5"),   string("SHA1"), 
                           string("JENKINS"),  string("HSIEH"), string() };
 
-    char (*pfunc)(char) = reinterpret_cast<char(*)(char)>(static_cast<int(*)(int)>(toupper));
-    transform(str.begin(), str.end(), str.begin(), pfunc);
```

Comment:
Not sure this would necessarily work on a big endian architecture, and less straight-forward than a lambda func in any case.